### ISSUE TITLE
Access violation exception when calling saveToFile 

### ISSUE THREAD

**Issue by martin-hackenberg-paradoxcat** [GENIVI/ramses#48]
Ramses Version: 26.0.4

Access violation Exception is thrown: Access violation reading location 0xFFFFFFFFFFFFFFFF.
Steps to reproduce:

1. Create scene
2. Create Effect in scene
3. setName on Effect
4. destroy Effect
5. call saveToFile on Scene

```
#include <gtest/gtest.h>

#include <ramses-framework-api/RamsesFramework.h>
#include <ramses-client-api/RamsesClient.h>
#include <ramses-client-api/Scene.h>
#include <ramses-client-api/Effect.h>
#include <ramses-client-api/EffectDescription.h>
#include <filesystem>

`constexpr char* emptyVertexShader =
	"#version 300 es\n\
		precision mediump float;\n\
		void main() {}";

constexpr char* emptyFragmentShader =
	"#version 300 es\n\
		precision mediump float;\n\
		void main() {}";

TEST(ramses, saveToFile_destroyedEffect) {
	ramses::RamsesFramework ramsesFramework;
	ramses::RamsesClient& client = *ramsesFramework.createClient("example client");
	ramses::Scene* scene = client.createScene(ramses::sceneId_t(123u), ramses::SceneConfig(), "export scene");

	ramses::EffectDescription desc{};
	desc.setVertexShader(emptyVertexShader);
	desc.setFragmentShader(emptyFragmentShader);
	ramses::Effect* effect = scene->createEffect(desc, ramses::ResourceCacheFlag_DoNotCache);
	effect->setName("SomeName");
	scene->destroy(*effect);

	scene->saveToFile("someFileName.ramses", false);

	ASSERT_TRUE(std::filesystem::remove("someFileName.ramses"));
}

**Comment by tohammer** [comment]
Thanks for the report! We can reproduce the issue and are working on a fix.

**Comment by tohammer** [comment]
It should be fixed with new 26.0.6 release: https://github.com/GENIVI/ramses/releases/tag/26.0.6
